apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -178,65 +231,58 @@ fn handle_command(device: &mut Device, data: &[u8]) -> Result<(), ParseError> {
         }
     }
 
-    return send_key(device, key);
+    send_key(device, key)
```

Comment:
Rust-ism: no final `return` when you can do it implicitly (flagged by `cargo clippy -- -W clippy::pedantic`).